installer/manifests/keptn/keptn-ingress.yaml

```diff
@@ -0,0 +1,26 @@
+---
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  annotations:
+    kubernetes.io/ingress.class: ingress.placeholder
+  name: keptn-ingress
+  namespace: keptn
+spec:
+  tls:
+    - hosts:
+        - '.keptn.domain.placeholder'
```

Comment:
Just to be 100% sure on this:

This used to be
```yaml
spec:
  tls:
    - hosts:
        - domain.placeholder
      secretName: sslcerts
```

Now it starts with a `.` and it is limited to the `keptn` namespace.

Reply:
Corrected. This was a c&p error.